src/main/java/Duke.java

```diff
@@ -13,29 +13,49 @@ public static void main(String[] args) {
         System.out.println("What can I do for you?");
         System.out.println("================================================");
 
-        String[] tasks = new String[100];
+        Task[] tasks = new Task[100];
         int i = 0;
+        int marker;
```

Comment:
Could change the naming of marker variable to something more comprehensive like taskStartIndex